firstvoices/backend/models/dictionary.py

```diff
@@ -259,3 +263,26 @@ class Meta:
 
     def __str__(self):
         return f"{self.category} - {self.dictionary_entry}"
+
+
+class WordOfTheDay(BaseSiteContentModel):
+    """
+    Table for word-of-the-day containing word and its respective date it was chosen to be wotd.
+    """
+
+    date = models.DateField(db_index=True)
+    dictionary_entry = models.ForeignKey(
+        "DictionaryEntry", on_delete=models.CASCADE, related_name="wotd_set"
```

Comment:
Please suggest if "cascade" works for on_delete or we should switch to some other option ?

Reply:
It seems like the correct solution to me, on delete of a dictionary entry the corresponding wotd model is also deleted.

Reply:
Agree